File: Models/Models/ViewModels/UserInfoViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CorporateQnAModels.Models.ViewModels
{
    public class UserInfoViewModel
    {
        public string Id { get; set; }
        public string FullName { get; set; }
        public string ImageUrl { get; set; }
        public string Designation { get; set; }
        public string Company { get; set; }
        public int NoOfLikes { get; set; }
        public int NoOfDislikes { get; set; }
        public int QuestionsAsked { get; set; }
        public int QuestionsAnswered { get; set; }
        public int QuestionsSolved { get; set; }

    }
}

File: CorporateQnA/Client/src/app/models/userInfoView.model.ts
export class UserInfoView {
     id: string;
     fullName: string;
     designation: string;
     company: string;
     imageUrl: string;
     noOfLikes: number;
     noOfDislikes: number;
     questionsAsked: number;
     questionsAnswered: number;
     questionsSolved: number;

     constructor(args: any) {
          this.id = args.id;
          this.fullName = args.name;
          this.designation = args.designation;
          this.company = args.company;
          this.imageUrl = args.imageUrl;
          this.noOfLikes = args.noOfLikes;
          this.noOfDislikes = args.noOfDislikes;
          this.questionsAsked = args.questionsAsked;
          this.questionsAnswered = args.questionsAnswered;
          this.questionsSolved = args.questionsSolved;
     }
}